src/ext/sse.js

```diff
@@ -116,50 +117,57 @@ This extension adds support for Server Sent Events to htmx.  See /www/extensions
 			return null;
 		}
 
-		var internalData = api.getInternalData(elt);
-
 		// get URL from element's attribute
-		var sseURL = api.getAttributeValue(elt, "sse-connect");
+		var sseURL = api.getAttributeValue(elt, "sse-connect") || getLegacySSEURL(elt);
```

Comment:
This appears to be an issue with the legacy implementation as well... I'll fix it for both in the extension but it will create an inconsistency in the behavior of legacy swapping handled by the extension and handled by htmx itself. 

I'm willing to fix it in htmx itself, but that seems like it significantly increases the scope of this pr beyond the sse extension. Let me know if you feel I should, or if I should make legacy handling in the extension mirror the existing legacy handling.

Reply:
We already have such inconsistency with legacy WS anyway, so I say, you may leave legacy SSE as it is